### PR TITLE
Nemesis updates 2

### PR DESCRIPTION
**Pull request by coderwilson**
May include other changes in order to get Nemesis working again.

### CODE REVIEW

**Comment by orkaboy** [comment]
You probably didn't intend to add the `area/__pycache__` files.

File: memory/main.py [hunk 1]
```diff
@@ -948,8 +948,8 @@ def item_address(num):
 
 def get_items_order():
     items = []
-    for x in range(100):
-        items.append(process.read_bytes(item_address(x), 1))
+    for x in range(150):
+        items.append(process.readBytes(item_address(x), 1))
```
Comment (jlward) [inline memory/main.py:952]: All of these should be updated to snake case. Otherwise they are not going to work.

**Comment by jlward** [comment]
Looks like we need to fix some camelCase -> snake_case and also remove some pyc files.

File: nemesis/arenaPrep.py [hunk 2]
```diff
@@ -1214,6 +1178,51 @@ def kilika_final_shop():
     xbox.tap_right()
     xbox.tap_b()  # Sell equipment
     menu.sell_all()
+    memory.main.wait_frames(10)
+    xbox.tap_a()
+
+def kilika_final_shop():
```
[the remark this answers is not in the document]
Comment (coderwilson) [reply nemesis/arenaPrep.py:1184]: it just got moved down, should be fine.

**Pull request merged**